ModernFlyouts/Controls/CrossFadeContentControl.cs

```diff
@@ -48,9 +49,11 @@ public class CrossFadeContentControl : ContentControl
         /// template part name.
         /// </summary>
         private const string SecondTemplatePartName = "SecondContentPresenter";
-        #endregion
+
+        #endregion Constants and Statics
```

Comment:
Could you just revert this?
It's hard to keep adding the same text on the start and end of a region. It requires both titles to be same. Please?